bin/lolcommits

```diff
@@ -408,10 +332,10 @@ if not (Choice.choices[:enable] || Choice.choices[:disable])
   elsif Choice.choices[:last]
     do_last
   elsif Choice.choices[:browse]
-    die_if_not_git_repo!
-    Launchy.open configuration.loldir
+    Fatals.die_if_not_git_repo!
+    Launcher.open_folder(configuration.loldir)
   elsif Choice.choices[:gif]
-    do_gif
+    HistoryAnimator.new(configuration).run(Choice.choices[:gif])
```

Comment:
Again, how about putting the `Choice.choices[:gif]` into the configuration object here (rather than passing it to run) and using it within the `HistoryAnimator`..  Also, maybe its just me but `HistoryAnimator` could maybe have a better name, how bout; `AnimatedTimelapseGif` or just `TimelapseGif`..